src/main/resources/db/changelog/changes/04-insert-books.yaml

```diff
@@ -0,0 +1,60 @@
+databaseChangeLog:
+  - changeSet:
+      id: insert-programming-books
+      author: Volodymyr
+      changes:
+        - insert:
+            tableName: books
+            columns:
+              - column: { name: title, value: "Clean Code" }
+              - column: { name: author, value: "Robert C. Martin" }
+              - column: { name: isbn, value: "9780132350884" }
+              - column: { name: price, value: "39.99" }
+              - column: { name: description, value: "A Handbook of Agile Software Craftsmanship" }
+              - column: { name: cover_image, value: "cleancode.jpg" }
+              - column: { name: is_deleted, valueBoolean: false }
+              - column: { name: user_id, valueNumeric: 1 }
```

Comment:
What is your business logic? you can explain it in the chat on the platform

Reply:
Seed data is used to avoid manual book creation during testing:
- `description` – for book details page;
- `cover_image` – static image filename;
- `is_deleted` – soft delete flag;
- `user_id` – admin (ID 1) for now, will be dynamic later.